src/pages/home/report/ReportActionsView.js

```diff
@@ -283,7 +283,7 @@ class ReportActionsView extends React.Component {
      */
     loadMoreChats() {
         // Only fetch more if we are not already fetching so that we don't initiate duplicate requests.
-        if (this.props.report.isLoadingMoreReportActions) {
+        if (this.props.report.isLoadingMoreReportActions || this.props.network.isOffline) {
```

Comment:
Why are we preventing this request from happening?

Reply:
Discussed above, I will get rid of it.

Reply:
Ok I think this is also related to the weird issue where we are setting "loading" states when no "loading" is actually happening and we should pause until there is agreement about how to proceed with that.